src/overlay/main.scss

```diff
@@ -1,6 +1,10 @@
 @import "../core/index-noreset.scss";
 
 .#{$css-prefix}overlay-wrapper {
+    position: absolute;
+    top: 0;
+    left: 0;
+
     .#{$css-prefix}overlay-inner {
```

Comment:
for those who use theme package, they won't get this change unless upgrade the theme package on config platform. If there is a proper way to solve the issue without changing the scss, use that. If not,  change this pr's target branch to feat/1.16.0

Reply:
I will change to feat/1.16.0 because I cannot think of another solution. If we do not have `position: absolute`, we would need to specify the parent container and place it there instead of body, but that would still have a similar issue depending on that parent's styles.